conans/client/detect.py

```diff
@@ -212,7 +212,8 @@ def _detect_os_arch(result, output):
                      'i86pc': 'x86',
                      'amd64': 'x86_64',
                      'aarch64': 'armv8',
-                     'sun4v': 'sparc'}
+                     'sun4v': 'sparc',
+                     '00f768d14c00': 'ppc64'}
```

Comment:
`platform.machine()` returns `00f768d14c00`? looks very random

Reply:
That's the return value.
Maybe python platform needs to be fixed first?

Reply:
Fixing python platform would take lot of time, and wouldn't be backwards compatible with many python versions people are already using, so we would need a workaround here.

I would say that ``00f768d14c00`` is some magic or random number of those machines, but maybe it is not the same in other AIX machines. In platforms with uname, ``uname -m`` should give the same result. Is that number the result, always in different AIX machines?